npm-debug.log

```diff
@@ -0,0 +1,37 @@
+0 info it worked if it ends with ok
```

Comment:
Cool. I'll remove it. Does the rest look okay? Would you change the style or implementation?